thesis/dependencies.tex

```diff
@@ -0,0 +1,13 @@
+\chapter{Dependencies}
+The project utilizes several external dependencies on both the front and back-end applications.
+They are managed with the package manager \glsi{npm} for \glsi{node}.
+
+All but one of these dependencies are \gls{oss} and free to use.
+They do hold different licenses on the source code which can be of special concern, why they are listed in the documentation. 
+This project does however not use any dependencies that require special steps.
+
+One dependency is a commercial product and there by not free to use, namely the \verb+nativescript-telerik-ui-pro+ package.
+The costs of the package is \$199 \citep{dependencies:nativescript}, i.e. kr. 1395, per developer, but is provided by Telerik free of charge for this project on a student license.
+
+The project utilizes only a subset of this package and, by this, may be to costly for the final product.
+To lower the costs an effort could be put into finding alternative dependencies that cover the use cases but do not require the payment.
```

Comment:
To avoid this cost, an effort could be made into finding an alternative product that can provide the same functionality, but without the commercial license.

eller noget i den stil. Kunne hvertfald godt formuleres bedre tænker jeg :)